libs/common/src/interfaces/issuance.interface.ts

```diff
@@ -15,4 +15,15 @@ export interface IIssuedCredential {
     previousPage: number;
     lastPage: number;
     data: IIssuedCredentialResponse[];
+  }
+  interface CredentialData {
```

Comment:
We are using the I prefix for interface names

Reply:
I am not exporting the `CredentialData` it's for internal use